renderer/webtorrent.js

```diff
@@ -73,23 +72,20 @@ function init () {
 
 // Starts a given TorrentID, which can be an infohash, magnet URI, etc. Returns WebTorrent object
 // See https://github.com/feross/webtorrent/blob/master/docs/api.md#clientaddtorrentid-opts-function-ontorrent-torrent-
-function startTorrenting (torrentSummary) {
```

Comment:
We can't use `torrentSummary` as parameter here, because we don't have one when adding new torrent. It is created after `wt-infohash` message.